docs/hoon/library/2k.md

```diff
@@ -8,38 +8,41 @@ navhome: /docs/
 
 Queue operations
 
-Container arm for queue operation arms. The contained arms inherit its
-sample `++qeu` `a`.
+Container arm for queue operation arms. The contained arms inherit `a`, its
+sample queue.
 
-Accepts
--------
+#### Accepts
 
-`a` is a queue, ++qeu.
+`a` is a tree.
 
-Source
-------
+#### Source
 
-    ++  to                                                  ::  queue engine
-      |/  a/(qeu)
+    ++  to                                                 
+      |_  a=(tree)
 
+#### Discussion
+
+`tree` is the generic tree-shape mold. Maps, sets, and queues all share that
```

Comment:
`+tree` is the generic mold builder for trees where each node looks like either `~` or `[n=<some value> l=tree r=tree]`.